secrets/hashicorpvault/hashicorpVault.go

```diff
@@ -0,0 +1,186 @@
+package hashicorpvault
```

Comment:
Don't use camelCase names for files, it's hashicorp_vault.go not hashicorpVault.go

Reply:
Standardized the naming 👍 

8792149